### ISSUE TITLE
CL has KeyError: 'CARO:0000000'

### ISSUE THREAD

**Issue by quirinmanz** [althonos/pronto#156]
Hi, 

Overall, I find this package pretty useful.

I tried to load the Cell Ontology as described in the Examples:

`cl = Ontology("http://purl.obolibrary.org/obo/cl.obo")`

Unfortunately, I get the following error:
![image](https://user-images.githubusercontent.com/50946714/154323705-a8cac4bf-7f07-4329-ab4f-eeff86e382e3.png)

It looks like `is_a` relationships are only expected from the same ontology.

Any help would be appreciated. 

Best,
Quirin

**Comment by althonos** [comment]
This is an problem on the CL side, I'll get in touch with the developers. It is not expected for an OBO ontology to contain dangling clauses (referencing undeclared entities). In the meantime, you can try using the `cl-basic.obo` release, which is lacking some axioms and relationships to other ontologies, but doesn't have this problem:
```python
>>> import pronto
>>> cl = pronto.Ontology("http://purl.obolibrary.org/obo/cl-basic.obo")
```

**Comment by quirinmanz** [comment]
Thank you, that is very helpful!

Interestingly, I get the following error, when using `from_obo_library`:
![image](https://user-images.githubusercontent.com/50946714/154437946-a2e8b656-56f6-4fd5-b7d5-5a6035be9721.png)
I have the same problem with other local obo files (not cl):
![image](https://user-images.githubusercontent.com/50946714/154438072-f64f3abe-914e-4c03-b158-127281c34704.png)

Do you have an idea on how to solve this?

**Comment by althonos** [comment]
You need to specify the extension with `Ontology.from_obo_library`:
```python
cl = pronto.Ontology.from_obo_library("cl-basic.obo")
```
and there is also a basic uberon (it will raise some warnings):
```python
uberon = pronto.Ontology.from_obo_library("uberon/basic.obo")
```

**Comment by cmungall** [comment]
I agree these ontologies should be fixed as its confusing to have these dangling classes - but it's not a violation of the spec, and it can be useful to be able to parse these
